README.md

```diff
@@ -2,6 +2,8 @@
 
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses/branch/main/graph/badge.svg?token=LvEIQ6tYti)](https://codecov.io/gh/ucsb-cs156-w22/team04-w22-5pm-courses)
 
+Heroku QA App Link: https://s22-4pm-2-courses-qa.herokuapp.com/
```

Comment:
I would also label the links as 4pm-2 dashboard and 4pm-2 deployment or something along those lines.

Reply:
I added that!